LOINC/manual_work/loinc_refresh.sql

```diff
@@ -242,4 +242,9 @@ AND crm.concept_code_1 = m.source_code AND crm.vocabulary_id_1 = m.source_vocabu
 AND crm.concept_code_2 = m.target_concept_code AND crm.vocabulary_id_2 = m.target_vocabulary_id
 AND crm.relationship_id = m.relationship_id
 AND crm.invalid_reason IS NOT NULL
-;
\ No newline at end of file
+;
```

Comment:
These changes are not needed

Reply:
I need this, because often I need to change cm